.gitignore

```diff
@@ -11,6 +11,11 @@
 
 .ipynb_checkpoints/
 
+SRH_testingnotebook.ipynb
```

Comment:
Since these files are specific to your working environment, I'd rather not include them in our `.gitignore`. You can eliminate the commit when rebasing to squash the rest.

My recommendation to avoid dealing with these extra files is to create a directory outside the metpy respository (mine is `notebook-sandbox`) and play with your notebooks there. That way you're not contaminating your local copy of the repo.